receiver/elasticsearchreceiver/metadata.yaml

```
@@ -950,3 +966,28 @@ metrics:
       value_type: int
     attributes: [document_state, index_aggregation_type]
     enabled: false
+  elasticsearch.process.cpu.usage:
+    description: CPU usage in percent, or -1 if not known at the time the stats are computed.
```

```suggestion
    description: CPU usage in percent.
```
Please don't report the metric if not known.